docs/docs/reference/changed/eta-expansion.md

```diff
@@ -3,42 +3,19 @@ layout: doc-page
 title: "Automatic Eta Expansion"
 ---
 
-Previously, a method reference `m` was converted to a function value
-only if the expected type was a function type. If that was not the
-case, one had to write `m _` to force the conversion (which is called
-eta-expansion).
+The conversion of _methods_ into _functions_ has been improved and happens automatically for methods with one or more parameters.
 
-For methods with one or more parameters, this restriction has now been
-dropped. Example:
-
-    def m(x: Boolean, y: String)(z: Int): List[Int]
-    val f1 = m
-    val f2 = m(true, "abc")
+```scala
+def m(x: Boolean, y: String)(z: Int): List[Int]
+val f1 = m
+val f2 = m(true, "abc")
+```
 
 This creates two function values:
+```scala
+f1: (Boolean, String) => Int => List[Int]
+f2: Int => List[Int]
+```
 
-    f1: (Boolean, String) => Int => List[Int]
-    f2: Int => List[Int]
-
-The syntax `m _` is no longer needed and will be deprecated in the
```

Comment:
I would leave that on the front page - it's important to know for users, and not so much a spec detail.